Documentation/ColumnsConfig/Type/Color/Index.rst

```diff
@@ -19,6 +19,12 @@ Color
 
 The TCA type :php:`color` should be used to render a JavaScript-based color picker.
 
+..  versionadded:: 13.0
+
+:ref:`Color palettes <t3tsconfig:pagecolorpalettes>` can be defined via
+:ref:`page TSconfig <t3tsconfig:setting-page-tsconfig>`. This way, for example,
+colors defined in a corporate design can be made accessible in an easy way.
```

Comment:
I don't really get the last sentence?

Reply:
The user can just click on the color instead of enter a RGB value manually.

Reply:
colors [...] can be made accessible could also be interpreted as "barrierefrei gemacht werden" this is what caused by confugsion. @sarahmccarthy123 you have an idea of how to rephrase this?

Reply:
Maybe "can be selected by a simple click" rather than "can be made accessible in an easy way"